### ISSUE TITLE
"No uninstallers were selected" bug

### ISSUE THREAD

**Issue by S197Stang** [Klocman/Bulk-Crap-Uninstaller#468]
Hello, i am very happy with this software but from time to time, when i try to uninstall a program, i get error that says there were no uninstallers selected.
![Untitled](https://user-images.githubusercontent.com/127688604/224565464-daa994f6-9d0e-4407-9001-92f66243ff73.jpg)
 Kind of annoying bug.

**Comment by Klocman** [comment]
Are you sure you didn't deselect items on the list by accident? If you use checkboxes make sure that they are checked.

**Comment by S197Stang** [comment]
Yes, i am sure. i  check item with checkbox. It does not happen all the time, just sometimes.

**Comment by S197Stang** [comment]
![Untitled](https://user-images.githubusercontent.com/127688604/226116126-57db9233-8c41-48e6-bc37-5d1a38d4a869.jpg)

Here is another screenshot with checked app. Also when i tried to uncheck that app, i could not do it. Other programs i can check and uncheck, but not that i checked previously.

**Comment by Klocman** [comment]
I can see the issue randomly pop up when messing with the groups (e.g. clicking on the group to select all, then selecting some items), but I couldn't find what exactly was wrong. It looks like a regression caused by moving to .NET 5/6.

**Comment by fadihanna123** [comment]
Same problem for me.

**Comment by Dorennor** [comment]
Same problem for me, can reproduce it quite often.
